.github/workflows/database.yml

```diff
@@ -20,94 +20,13 @@ jobs:
       run:
         shell: bash -l {0}
 
-    env:
-      ENV_FILE: ci/deps/actions-37-locale.yaml
-      LOCALE_OVERRIDE: zh_CN.UTF-8
-
-    services:
-      mysql:
-        image: mysql
-        env:
-          MYSQL_ALLOW_EMPTY_PASSWORD: yes
-          MYSQL_DATABASE: pandas
-        options: >-
-          --health-cmd "mysqladmin ping"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 3306:3306
-
-      postgres:
-        image: postgres
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: pandas
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 5432:5432
-
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v1
-
-    - name: Cache conda
-      uses: actions/cache@v1
-      env:
-        CACHE_NUMBER: 0
-      with:
-        path: ~/conda_pkgs_dir
-        key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
-          hashFiles('${{ env.ENV_FILE }}') }}
-
-    - uses: conda-incubator/setup-miniconda@v2
-      with:
-        activate-environment: pandas-dev
-        channel-priority: strict
-        environment-file: ${{ env.ENV_FILE }}
-        use-only-tar-bz2: true
-
-    - name: Build Pandas
-      uses: ./.github/actions/build_pandas
-
-    - name: Test
-      run: ci/run_tests.sh
-      if: always()
-
-    - name: Build Version
-      run: pushd /tmp && python -c "import pandas; pandas.show_versions();" && popd
-
-    - name: Publish test results
-      uses: actions/upload-artifact@master
-      with:
-        name: Test results
-        path: test-data.xml
-      if: failure()
-
-    - name: Print skipped tests
-      run: python ci/print_skipped.py
-
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
-      with:
-        flags: unittests
-        name: codecov-pandas
-        fail_ci_if_error: false
-
-  Linux_py37_cov:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash -l {0}
+    strategy:
+      matrix:
+        ENV_FILE: [ci/deps/actions-37-locale.yaml, ci/deps/actions-37-cov.yaml]
```

Comment:
Hi @fangchenli - in #39758 you wrote

> and remove the actions-37-cov build if it's no longer required.

but here it looks like you're keeping it - is it still required then?

Reply:
One of them is testing the minimum support versions of SQL-related packages. So I think it's better to keep both of them. But should rename them to reflect their actual purpose.